### PR TITLE
Query param validations

### PR DESCRIPTION
**Pull request by surekhag**

### CODE REVIEW

File: server/app/api/controllers/projects.js [hunk 1]
```diff
@@ -59,7 +59,8 @@ module.exports = {
   },
   getAll: function(req, res, next) {    
     //todo - remove or optimize it
-     const query = req.query? req.query: {}
+    const status = req.query.status
```
Comment (bhushan-oe) [inline server/app/api/controllers/projects.js]: Please use spread operator

File: server/app/api/controllers/projects.js [hunk 2]
```diff
@@ -59,7 +59,8 @@ module.exports = {
...
+    const query = status ? {status} : {}
```
Comment (bhushan-oe) [inline server/app/api/controllers/projects.js]: no need of this variable

File: server/app/api/controllers/users.js [hunk 3]
```diff
@@ -116,8 +116,9 @@ module.exports = {
     });
   },
   getAll: function (req, res, next) {
-    const query = req.query || {}
-    userModel.find(query, function (err, users) {
+    const status = req.query.status
```
Comment (bhushan-oe) [inline server/app/api/controllers/users.js]: same comments as project file

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

File: server/app/api/controllers/projects.js (hunk 1 above)
Comment (surekhag) [reply server/app/api/controllers/projects.js]: Done

File: server/app/api/controllers/projects.js (hunk 2 above)
Comment (surekhag) [reply server/app/api/controllers/projects.js]: Removed.

File: server/app/api/controllers/users.js (hunk 3 above)
Comment (surekhag) [reply server/app/api/controllers/users.js]: Updated.

File: server/app/api/controllers/projects.js [hunk 4]
```diff
@@ -57,11 +57,9 @@ module.exports = {
       }
     });
   },
-  getAll: function(req, res, next) {    
-    //todo - remove or optimize it
-    const status = req.query.status
-    const query = status ? {status} : {}
-    projectModel.find(query, function(err, projects) {
+  getAll: function(req, res, next) { 
+    let {status} = req.query;
```
Comment (bhushan-oe) [inline server/app/api/controllers/projects.js]: why let? status value is not changing...

File: server/app/api/controllers/users.js [hunk 5]
```diff
@@ -116,9 +116,8 @@ module.exports = {
     });
   },
   getAll: function (req, res, next) {
-    const status = req.query.status
-    const query = status ? {status} : {}
-   userModel.find(query, function (err, users) {
+    let {status} = req.query;
```
Comment (bhushan-oe) [inline server/app/api/controllers/users.js]: why let?

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

File: server/app/api/controllers/users.js (hunk 5 above)
Comment (surekhag) [reply server/app/api/controllers/users.js]: Missed . Updated now.

File: server/app/api/controllers/projects.js (hunk 4 above)
Comment (surekhag) [reply server/app/api/controllers/projects.js]: Missed . Updated now.

**Comment by surekhag** [comment]
Merge with PR #43. Closing it.

**Pull request closed without merging**